cli/src/commands/squash.rs

```diff
@@ -37,13 +37,12 @@ use crate::ui::Ui;
 /// For example, `jj squash --into @--` moves changes from the working-copy
 /// commit to the grandparent.
 ///
-/// If, after moving changes out, the source revision is empty compared to its
-/// parent(s), it will be abandoned. Without `--interactive`, the source
-/// revision will always be empty.
+/// If, `--interactive` is not set, no paths are provided, and `--keep-empty`
+/// is not provided, after moving changes out, the source will be abandoned.
```

Comment:
rewording nit:

```suggestion
/// If `--interactive` is not set, no paths are provided, and `--keep-empty`
/// is not provided, the source will be abandoned.
```

Feel free to rephrase as necessary.